docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc

```diff
@@ -19,10 +19,19 @@ A Red Hat account is required in order to access the user pull secret.
 .Procedure
 
 * Start the {prod} virtual machine:
+
+** For native hypervisors:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} start
+----
+
+** For VirtualBox, the `--vm-driver virtualbox` and `--bundle` flags are required:
 +
 [subs="+quotes,attributes"]
 ----
-$ {bin} start -b _path_to_system_bundle_
+$ {bin} start --vm-driver virtualbox --bundle _path_to_system_bundle_
```

Comment:
@gbraad @robin-owen should we go with `path_to_system_bundle` or `path_to_virtualbox_bundle` , I am not sure if it is replaced by some variable?

Reply:
I think this is OK